modules/core/04-channel/keeper/upgrade.go

```diff
@@ -74,8 +74,8 @@ func (k Keeper) ChanUpgradeTry(
 	proposedConnectionHops []string,
 	counterpartyUpgradeFields types.UpgradeFields,
 	counterpartyUpgradeSequence uint64,
-	proofCounterpartyChannel,
-	proofCounterpartyUpgrade []byte,
+	counterpartyChannelProof,
+	counterpartyUpgradeProof []byte,
```

Comment:
my personal opinion is that we should never have to prefix any `proof []byte` with `counterparty`, because its implicit that the proof is for a counterparty state machine?

Reply:
I removed it [here](https://github.com/cosmos/ibc-go/pull/5599/commits/4293dd45481e2e13f8337a74d554477cea5f750d).